src/Uno.UI/Themes/WinUI/Resources/PriorityDefault/NavigationView.xaml

```diff
@@ -0,0 +1,731 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:primitives="using:Microsoft.UI.Xaml.Controls.Primitives"
+    xmlns:local="using:Microsoft.UI.Xaml.Controls">
+
+    <Style TargetType="local:NavigationView">
```

Comment:
In current Uno, this style contains the VisibleBounds padding. I'm wondering if we should add it there as well.

Reply:
Yeah that would make sense. It won't matter for now, the `NavigationView` style is excluded because of the dependence on `ItemsRepeater`.